anymail/webhooks/mailgun.py

```diff
@@ -10,6 +10,14 @@
 from ..signals import tracking, AnymailTrackingEvent, EventType, RejectReason
 from ..utils import get_anymail_setting, combine
 
+# Use safer comparison method if available
+# https://docs.python.org/2/library/hmac.html#hmac.compare_digest
+if hasattr(hmac, "compare_digest"):
+    compare_digest = hmac.compare_digest
+else:
+    def compare_digest(s1, s2):
+        return s1 == s2
+
```

Comment:
I'd suggest instead switching to [`django.utils.crypto.constant_time_compare`](https://github.com/django/django/blob/stable/1.8.x/django/utils/crypto.py#L79-L84), which is available on all Django versions we support. That preserves the security aspect of the code (unlike a simple `==` compare).